src/app/components/OperatorsList/OperatorsList.tsx

```diff
@@ -63,53 +72,43 @@ const OperatorsList = () => {
     return (operators || []).map((operator: any) => {
       const data = [
         <Link href={`${config.routes.OPERATORS.HOME}/${operator.id}`} className={classes.Link}>
-          <Grid item className={classes.OperatorLogo} style={{ backgroundImage: operator.logo ? `url(${operator.logo})` : '' }} />
-          {operator.name}
-          <OperatorType type={operator.type} />
-        </Link>,
-        <Link href={`${config.routes.OPERATORS.HOME}/${operator.id}`} className={classes.Link}>
-          <Box component="div" display={{ xs: 'block', sm: 'block', md: 'none', lg: 'none' }}>
-            {operator.id}
-          </Box>
-          <Box component="div" display={{ xs: 'none', sm: 'none', md: 'block', lg: 'block' }}>
-            {operator.id}
-          </Box>
+          <OperatorDetails operator={operator} />
         </Link>,
         <Box component="div" display={{ xs: 'block', sm: 'block', md: 'block', lg: 'block' }}>
           <Status entry={operator} />
         </Box>,
-        <Link href={`${config.routes.OPERATORS.HOME}/${operator.id}`} className={classes.Link}>
-          {operator.validators_count}
-        </Link>,
       ];
 
       const performances = getPerformances(operator.performance);
       for (let i = 0; i < performances.length; i += 1) {
         const performance = performances[i];
-        data.push(
-          <Link href={`${config.routes.OPERATORS.HOME}/${operator.id}`} className={classes.Link}>
-            {`${parseFloat(String(performance.value)).toFixed(2)}%`}
-          </Link>,
-        );
+        if (performance.key === '24h') {
```

Comment:
You can take it as `operator.performance['24h'] || 0` directly, without these cycles and usage of `getPerformance` helper.

Reply:
done